static/app/utils/customMeasurements/customMeasurementsProvider.tsx

```diff
@@ -59,17 +63,16 @@ export function CustomMeasurementsProvider({
           return;
         }
 
-        const newCustomMeasurements = Object.keys(response).reduce<MeasurementCollection>(
-          (acc, customMeasurement) => {
-            acc[customMeasurement] = {
-              key: customMeasurement,
-              name: customMeasurement,
-            };
-
-            return acc;
-          },
-          {}
-        );
+        const newCustomMeasurements = Object.keys(
+          response
+        ).reduce<CustomMeasurementCollection>((acc, customMeasurement) => {
+          acc[customMeasurement] = {
+            key: customMeasurement,
+            name: customMeasurement,
+            supportedFunctions: response[customMeasurement]?.functions ?? [],
```

Comment:
```suggestion
            supportedFunctions: response[customMeasurement].functions ?? [],
```

If `customMeasurement` is coming from `Object.keys(response)`, then the key should always return a value, right? Unless the response can have `[key]: undefined` or something else 🤔 Is this possible?

Also if we know that there's always functions on this object, then I was thinking we can drop `?? []` too

Reply:
👍 It shouldnt be possible for the key to be undefined here and the measurements-meta endpoint is expected to always return a value. Updated!